Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -87,7 +87,7 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
         case .revampedShippingLabelCreation:
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .blazeCampaignObjective:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
I noticed that the previously selected `objective` was tracked and sent in the Blaze campaign creation request even after I set this as `false`. 

I think this will be an issue if we want to turn off the feature after the release. What do you think?

Reply:
Good point! I mitigated this by checking for the feature flag before initialize the campaign objective in 9cec245bbaf43e6cd62fedca094862b019f6a799.